packages/manager/apps/key-management-service/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { createHashRouter, RouterProvider } from 'react-router-dom';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import { odsSetup } from '@ovhcloud/ods-common-core';
+import { useShell } from '@ovh-ux/manager-react-shell-client';
+import '@ovhcloud/ods-theme-blue-jeans';
```

Comment:
You could move this import in the index where there is the import of another global.css